### ISSUE TITLE
CTD, apparently related to Pale Shadow spell summons

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#197]
Discord report

> Two CTDs in a row.  They seem to be to do with the spell Pale Shadow or perhaps the area I'm in (Shriekwind Bastion).
> Smart Harvest won't pull from bodies, no.  I changed that option right away.  All it grabs really is ingredients and some loose items.

[Crash_2020_8_29_3-39-3.txt](https://github.com/SteveTownsend/SmartHarvestSE/files/5145193/Crash_2020_8_29_3-39-3.txt)
[SmartHarvestSE.log](https://github.com/SteveTownsend/SmartHarvestSE/files/5145194/SmartHarvestSE.log)

**Comment by SteveTownsend** [comment]
Reproduced the crash as follows, even after removing summons and dynamic REFR or Base as followers.
```
J:\GitHub\SmartHarvestSE\src\WorldState\PartyMembers.cpp (82): shse::PartyMembers::AdjustParty
J:\GitHub\SmartHarvestSE\src\Looting\ScanGovernor.cpp (536): shse::ScanGovernor::DoPeriodicSearch
J:\GitHub\SmartHarvestSE\src\PluginFacade.cpp (249): shse::PluginFacade::ScanThread
J:\GitHub\SmartHarvestSE\src\PluginFacade.cpp (102): `shse::PluginFacade::Start'::`2'::<lambda_1>::operator()
```

**Comment by SteveTownsend** [comment]
There is another instance where Follower eligibility as to be checked. Testing again.

**Comment by SteveTownsend** [comment]
Now we see this, which means we skip the crashing code. Fixed for v2.9.2.0.
```
2020-08-30 16:32:19.692    trace  49276 REFR 0xff001fe7 is 339.96 units away, loot range 960.00 XY-units, 288.00 Z-units
2020-08-30 16:32:19.692    debug  49276 Actor Forsworn Looter/0xff001fe7 is teammate
2020-08-30 16:32:19.692    debug  49276 Actor summoned = false
2020-08-30 16:32:19.692    trace  49276 NPC Forsworn Looter/0xff001fe7 at distance 339.96 ineligible as Follower
```
